openssl/src/ssl/mod.rs

```diff
@@ -282,6 +320,12 @@ impl SslContext {
         }
     }
 
+    pub fn set_read_ahead(&self, m: c_long) {
+        unsafe {
+            ffi::SSL_CTX_set_read_ahead(*self.ctx, m);
```

Comment:
As a client, I'd prefer to have `m: usize` or `m: u32`, i.e. any rust type which is directly available without importing `libc::c_long`.